analysis/wwz/get_wwz_yields.py

```diff
@@ -562,23 +677,39 @@ def make_syst_plots(histo_dict,grouping_mc,grouping_data,save_dir_path,year):
             data_nom = merge_overflow(histo_grouped_data[{"systematic":"nominal"}])
 
             for syst in syst_var_lst:
-                if "btag" not in syst: continue
+                if syst not in JERC_LST: continue
+                #if "btag" not in syst: continue
                 #if "uncorrelated" not in syst: continue
                 #if "lepSF" not in syst: continue
                 #if "PreFiring" not in syst: continue
                 #if "PU" not in syst: continue
                 #if "ISR" not in syst and "FSR" not in syst: continue
                 #if "renorm" not in syst and "fact" not in syst: continue
 
-                # Skip the variations that don't apply (TODO: why are these in the hist to begin with??)
-                if year == "UL16APV": blacklist_years = ["2016","2017","2018","2022","2022EE"]
-                if year == "UL16": blacklist_years = ["2016APV","2017","2018","2022","2022EE"]
-                if year == "UL17": blacklist_years = ["2016APV","2016","2018","2022","2022EE"]
-                if year == "UL18": blacklist_years = ["2016APV","2016","2017","2022","2022EE"]
-                if year == "2022": blacklist_years = ["2016APV","2016","2017","2018","2022EE"]
-                if year == "2022EE": blacklist_years = ["2016APV","2016","2017","2018","2022"]
-                if year == "run3": blacklist_years = ["2016APV","2016","2017","2018"]
-                if year == "all": blacklist_years = []
+                if (("uncorrelated" in syst) or (syst.startswith("JER"))):
```

Comment:
Can you explain conceptually what this is changing?

Reply:
no clue, and I don't see why it is needed. I have removed it for now, but can always add this back in if for some reason it is helpful